test/topology_experimental_raft/test_coordinator_aborting.py

```diff
@@ -0,0 +1,163 @@
+import pytest
+import asyncio
+import logging
+import re
+import time
+from test.pylib.internal_types import ServerInfo
+
+from test.pylib.manager_client import ManagerClient
+from test.pylib.scylla_cluster import Cluster, ReplaceConfig
+from test.pylib.util import wait_for_cql_and_get_hosts, read_barrier
+from cassandra.cluster import SimpleStatement, ConsistencyLevel
+from test.topology.util import check_token_ring_and_group0_consistency
+
+
+logger = logging.getLogger(__name__)
+
+
+async def restart_coordinator(manager: ManagerClient, expr, coordinator: ServerInfo, 
+                              verification_node: ServerInfo, node_under_ops: ServerInfo):
+    log = await manager.server_open_log(node_under_ops.server_id)
+    await log.wait_for(expr, timeout=30)
+    await manager.server_stop(coordinator.server_id)
+    await manager.server_not_sees_other_server(verification_node.ip_addr, coordinator.ip_addr)
+    await manager.server_start(coordinator.server_id)
+
+
+@pytest.mark.asyncio
+async def test_kill_coordinator_during_op(manager: ManagerClient) -> None:
+    """ Kill cooridinator while topology operation running for cluster: decommission,
```

Comment:
space at the beginning of comment should be removed
`cooridinator` -> `coordinator`
`running` -> `is running`